backend/uid.py

```diff
@@ -0,0 +1,114 @@
+import time
+import threading
+import logging
+
+logger = logging.getLogger(__name__)
+logger.setLevel(level=logging.INFO)
+
+console_handler = logging.StreamHandler()
+console_formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+console_handler.setFormatter(console_formatter)
```

Comment:
add the file logger handler to get statistics about all the users.
maybe you can use `loguru` library, it is more ease of use.